frontend_vue/src/components/tokens/credit_card_v2/TokenDisplay.vue

```diff
@@ -1,24 +1,42 @@
 <template>
-	<div class="flex flex-col items-center">
-		<CreditCardToken
-			:token-data="tokenData" />
-		<base-button
-			class="mt-24"
-			@click="handleDownloadCC()">
-			Download Credit Card
-		</base-button>
-	</div>
+  <div class="flex flex-col items-center min-h-[448px] justify-center">
```

Comment:
I notice this looks a bit broken on mobile:
may we fix the alignments of items?
all input and buttons can take the whole line I think

<img width="329" alt="Screenshot 2024-10-30 at 15 02 11" src="https://github.com/user-attachments/assets/32856899-e4d1-4fc0-95fa-b9b9b5e7672a">